django_project/alerts/tasks.py

```diff
@@ -128,9 +134,12 @@ def process_alert(setting: AlertSetting, runner: AnalysisRunner):
             )
             if name is not None or value is not None:
                 if check_threshold(setting, value):
-                    trigger_alert(setting, value, name)
+                    trigger_alert(setting, value, name, None)
```

Comment:
default for the message parameter is already "", you don't explicitly pass it when unnecessary